ops/Dockerfile

```diff
@@ -1,5 +1,28 @@
-ARG UPSTREAM_IMAGE="amazonlinux"
-FROM $UPSTREAM_IMAGE
+# declaring flyway as a COPY source in al2 image
+FROM flyway/flyway as flyway
+RUN echo flyway.placeholders.type.int4=int4 > /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.type.text=text >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.tablespaces-escape=-- >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.drop-tablespaces-escape= >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.alter-column-type=type >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.hsql-only-alter=-- alter >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.psql-only-alter=alter >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.alter-rename-column=rename column >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.alter-rename-constraint=rename constraint >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.rename-to=to >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.index-create-concurrently=concurrently >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.sequence-start=start >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.sequence-increment=increment >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.perms= >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.psql-only= >> /flyway/conf/flyway.conf \
+    && echo flyway.placeholders.logic.hsql-only=-- >> /flyway/conf/flyway.conf \
+    && echo flyway.table=schema_version >> /flyway/conf/flyway.conf \
+    && echo flyway.schemas=public >> /flyway/conf/flyway.conf \
+    && echo flyway.url=jdbc:postgresql://db:5432/bfd >> /flyway/conf/flyway.conf \
+    && echo flyway.password=bfd >> /flyway/conf/flyway.conf \
+    && echo flyway.user=bfd >> /flyway/conf/flyway.conf
```

Comment:
I think it could make sense to just create the `flyway.conf` file outside of here and just copy the file in? Would make tracking changes and future edits a wee bit easier, maybe.

Reply:
You're not wrong, and I ain't married to this solution. I have some things in mind that just need be tidied up. I'll take your suggestion and add that to the list of things to do. Thanks!

Reply:
If you put it in a config file, The `DatabaseSchemaManager` could also read the props in from a file. (With maybe some filtering/overriding for anything that needs to be different/dynamic.